templates/4_Post_Clustering.ipynb

```diff
@@ -24,7 +24,8 @@
     "import numpy as np\n",
```

Comment:
<p>Looks like not quite all of the changes we discussed last week got included. </p><p>The code above is identifying cells in the immune_other category which express CD3 as strongly as CD4T and CD8Ts do. These are both Tcell populations, so the cells from immune_other which have high levels of CD3 should be assigned to a new category, tcell_other. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>